gen_template_docs.py

```diff
@@ -24,7 +24,8 @@
 amq_ssl_desc = None
 
 LINKS =  { "redhat-sso-cd-openshift:1.0": "../../templates/sso-cd-openshift{outfilesuffix}[`redhat-sso-7-tech-preview/sso-cd-openshift`]",
-           "redhat-sso-cd-openshift:5.0": "../../templates/sso-cd-openshift{outfilesuffix}[`redhat-sso-7-tech-preview/sso-cd-openshift`]"}
+           "redhat-sso-cd-openshift:5.0": "../../templates/sso-cd-openshift{outfilesuffix}[`redhat-sso-7-tech-preview/sso-cd-openshift`]",
+           "redhat-sso-cd-openshift:6": "../../templates/sso-cd-openshift{outfilesuffix}[`redhat-sso-7-tech-preview/sso-cd-openshift`]"}
```

Comment:
I would list both (IOW `6` and `6.0`) versions here. If you want to have two separate `6` and `6.0` tags in container catalog, the image to be resolvable / referencable by both `:6` and `:6.0` version, then it should have also two versions of the documentation (even when it would be identical for both versions).

Reply:
+1

Reply:
That's a good suggestion, thanks!